library/src/main/java/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -53,6 +54,10 @@ public class ChuckerInterceptor private constructor(
         decoders,
     )
 
+    init {
+        builder.context.createShortcut()
```

Comment:
I don't really like it enforced. IMO users should be able to opt–out of it with an XML boolean resource.

Reply:
Don't have strong preference here, but don't like to such init.

Reply:
I added a flag in builder to enable switch